robot/web/web.go

```diff
@@ -190,17 +190,17 @@ type Service interface {
 	// Start starts the web server
 	Start(context.Context, weboptions.Options) error
 
-	// Stop stops the main web service (but leaves lite running.)
+	// Stop stops the main web service (but leaves module service running.)
 	Stop()
 
-	// Start starts the web server
-	StartLite(context.Context) error
+	// StartModule starts the lightweight module service
```

Comment:
should use ModuleManager or ModuleServer. Also we don't need to call it lightweight anymore here